utils/pythonutils/utils.go

```diff
@@ -152,6 +158,50 @@ func getFilePath(srcPath, fileName string) (string, error) {
 	return filePath, nil
 }
 
+// Create the CmdOutputPattern objects that can capture group content that may span multiple lines for logs that have line size limitations.
+// Since the log parser parse line by line, we need to create a parser that can capture group content that may span multiple lines.
+func getMultilineSplitCaptureOutputPattern(startCollectingPattern, captureGroup, endCollectingPattern string, handler func(pattern *gofrogcmd.CmdOutputPattern) (string, error)) (parsers []*gofrogcmd.CmdOutputPattern) {
+	// Prepare regex patterns.
+	oneLineRegex := regexp.MustCompile(startCollectingPattern + `(` + captureGroup + `)` + endCollectingPattern)
```

Comment:
its dynamic patterns that are constructed by the given arguments to the method. I don't see any benefit to refactoring it outside because I need those values at the method